static/js/markdown.js

```diff
@@ -297,6 +297,29 @@ function handleAvatar(email) {
            ' title="' + email + '">';
 }
 
+function handleTimestamp(time) {
+    let timeobject;
+    if (isNaN(time)) {
+        // Momment throws a large deprecation warning when it has to fallback
+        // to the Date() constructor. We needn't worry here and can let bugdown
+        // handle any dates that moment misses.
+        moment.suppressDeprecationWarnings = true;
+        timeobject = moment(time); // not a Unix timestamp
+    } else {
+        // JavaScript dates are in milliseconds, Unix timestamps are in seconds
+        timeobject = moment(time * 1000);
+    }
+    const istimevalid = !(timeobject === null || !timeobject.isValid());
+
+    // Generate HTML
+    let timestring = '<span class="timestamp"';
+    if (istimevalid) {
+        timestring += ' value="' + timeobject.unix() + '"';
```

Comment:
`value` is not a valid attribute of `span`. We should use `data-value` or something.

Reply:
yeah, or `data-timestamp` might be clearer.